### ISSUE TITLE
add XMR donation address

### ISSUE THREAD

**Issue by fugbixer** [mchangrh/sb-mirror#19]
https://sponsor.ajay.app/donate/

projects like this can use bounties for fixing certain issues or
to refund instances, website costs etc.

therefore please add a XMR (Monero) address for donations since it's more privacy friendly

Thank you!

**Comment by mchangrh** [comment]
This is a third party implementation of a mirror, I think you mean ajayyy/SponsorBlockSite? 

Cc @ajayyy

**Comment by fugbixer** [comment]
no, i would like to be able to donate for this project as well.
since its good that you provide a method to run mirrors.
